uint8 control_mode
uint8 status
float32 pos_x
float32 pos_y
float32 yaw


